core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/json/JSONLScanSourceOpExec.scala

```diff
@@ -44,10 +38,14 @@ class JSONLScanSourceOpExec private[json] (
         }
 
         Tuple.newBuilder.add(schema, fields.toArray).build
-
+      }) match {
+        case Success(tuple) => tuple
+        case Failure(_)     => null
```

Comment:
similar comment with the csv source op exec

Reply:
done